lighthouse-core/gather/computed/trace-of-tab.js

```diff
@@ -45,18 +45,30 @@ class TraceOfTab extends ComputedArtifact {
       .filter(e => {
         return e.cat.includes('blink.user_timing') ||
           e.cat.includes('loading') ||
-          e.cat.includes('devtools.timeline') ||
-          e.name === 'TracingStartedInPage';
```

Comment:
think you don't want to filter this out to support the legacy case? Or in the fallback case need to search for the first `TracingStartedInPage` in the original `trace.traceEvents`

Reply:
also, does the filter now need to let through `e.name === 'TracingStartedInBrowser'` events?

Reply:
Both TracingStartedInPage and TracingStartedInBrowser are in devtools.timeline category which is already allowed.

Reply:
It seems like it was in toplevel once upon a time based on the fixture change below?

my traces from ~December have it in `disabled-by-default-devtools.timeline` so I guess it's a question of how far back we want to go

Reply:
AFAICT TracingStartedInPage was introduced 4 years ago with devtools.timeline category. See https://codereview.chromium.org/254613002/diff/90001/Source/core/inspector/InspectorTracingAgent.cpp
I wonder where the traces with it being in toplevel come from?

Reply:
Hm, no clue then 🤷‍♂️ 4 years is plenty good enough for me 😆